native/tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "baseUrl": ".",
     "isolatedModules": true,
     "jsx": "react",
+    "module": "es2020",
```

Comment:
Why this change? For the dynamic imports?

If yes, then are we actually sure that React-Native/Hermes supports this properly? I guess we are because it works :D

Reply:
Yes, because of dynamic imports.

Hmm yes, I think so, everything should work :D Did not look specifically whether it is supported by react native/hermes though